protocol/0004-AMND-amends.md

```diff
@@ -22,6 +22,12 @@ Start date: `2020-03-12` <br>
 - Attempting to alter details on a filled order will cause the amend to be rejected (<a name="0004-AMND-017" href="#0004-AMND-017">0004-AMND-017</a>)
 - Attempting to alter details on a cancelled order will cause the amend to be rejected (<a name="0004-AMND-018" href="#0004-AMND-018">0004-AMND-018</a>)
 - Attempting to alter details on an expired order will cause the amend to be rejected (<a name="0004-AMND-019" href="#0004-AMND-019">0004-AMND-019</a>)
+- Attempts to amend a liquidity provision order will be rejected (<a name="0004-AMND-020" href="#0004-AMND-020">0004-AMND-020</a>)
```

Comment:
You can amend LP provision orders, so this needs more precise formulation.

Reply:
Will remove
To be added to LP spec